src/library_openal.js

```diff
@@ -842,7 +842,7 @@ var LibraryOpenAL = {
       case {{{ cDefs.AL_VELOCITY }}}:
         return AL.currentCtx.listener.velocity;
       case {{{ cDefs.AL_ORIENTATION }}}:
-        return AL.currentCtx.listener.direction.concat(AL.currentCtx.listener.up);
+        return AL.currentCtx.listener.direction + AL.currentCtx.listener.up;
```

Comment:
These are arrays so this changes the result, doesn't it? E.g.

```
> [0, 0, 0] + [1, 2, 3]
'0,0,01,2,3'
> [0, 0, 0].concat([1, 2, 3])
[ 0, 0, 0, 1, 2, 3 ]
```

Reply:
oops, reverted.